api/transfer.go

```diff
@@ -0,0 +1,51 @@
+package api
+
+import (
+	"context"
+	"fmt"
+	"github.com/ovalfi/go-sdk/helpers"
+	"github.com/ovalfi/go-sdk/model"
+	"net/http"
+)
+
+const transferAPIVersion = "v1/transfer"
+
+// InitiateTransfer makes an API request using Call to initiate a transfer
+func (c *Call) InitiateTransfer(ctx context.Context, request model.InitiateTransferRequest) (model.Transfer, error) {
+	endpoint := fmt.Sprintf("%s%s", c.baseURL, transferAPIVersion)
+
+	fL := c.logger.With().Str("func", "InitiateTransfer").Str("endpoint", endpoint).Logger()
+	fL.Info().Msg("starting...")
+	fL.Info().Str("customerID", request.CustomerID).Str("reference", request.Reference).
+		Str("currency", request.Currency).Interface("destination", request.Destination).
+		Float64("amount", request.Amount).Str("note", request.Note).Str("reason", request.Reason).
+		Str("reference", request.Reference).
```

Comment:
you can just do `.Interface("request", request)`

Reply:
Updated